openassessment/xblock/static/js/spec/studio/oa_edit_assessments.js

```diff
@@ -250,4 +250,24 @@ describe("OpenAssessment edit assessment views", function() {
         it("Loads a description", function() { testLoadXMLExamples(view); });
         it("shows an alert when disabled", function() { testAlertOnDisable(view); });
     });
+
+    describe("OpenAssessment.EditStaffAssessmentView", function() {
+        var view = null;
+
+        beforeEach(function() {
+            view = new OpenAssessment.EditStaffAssessmentView();
```

Comment:
Spoke with @andy-armstrong about this. Since the Backbone code being tested doesn't rely on interacting with the DOM at all, I've removed the `element` reference here to make what is being tested more clear.